oat-sa/tao-core
# fix/AUT-2673/inline-figure-img

Related to: [AUT-2673](https://oat-sa.atlassian.net/browse/AUT-2673)

**Description:**
Display Image inline on Figure tag

**Changes:**

- https://github.com/oat-sa/tao-core-ui-fe/pull/528
- Bundle CSS package files

**Companion extensions:**
- https://github.com/oat-sa/extension-tao-mediamanager/pull/453
- https://github.com/oat-sa/extension-tao-itemqti/pull/2279

**How to check:**

On Assets

- Create an Asset with a text block. Add in the middle of the text an image. Play with the inline panel controls of the image to check if it is positioned correctly.
- Check Preview.

On Items

- Create an Item with a text block. Add in the middle of the text an image. Play with the inline panel controls of the image to check if it is positioned correctly.
- Add the previously created Asset with the Figure image on the inline position to check the position of the Asset image inside the Item.
- Check Preview

On Tests

- Check Terre Preview
- Delivery Terre

Check a test on Delivery.
